### PR TITLE
Comment validation

### PR DESCRIPTION
**Pull request by vrajmohan**

### CODE REVIEW

File: regulations/tasks.py [hunk 1]
```diff
@@ -31,6 +38,8 @@ def submit_comment(body):
                 'X-Api-Key': settings.REGS_API_KEY,
             }
         )
+        response.raise_for_status()
+        logger.info(response.text)
```
Comment (cmc333333) [inline regulations/tasks.py]: Quick check -- is there anything in this response that could be considered sensitive, or is it just the submission id, or similar?

File: regulations/tasks.py [hunk 2]
```diff
@@ -38,6 +47,13 @@ def build_comment(body):
 
 
 def extract_files(body):
+    '''
+    Extracts the files that are to be attached to the comment.
+    Returns a collection of dicts where for each dict:
+        - dict['key'] specifies the file to be attached from S3
+        - dict['name'] specifies the name under which the file is to be
+          attached.
+    '''
```
Comment (cmc333333) [inline regulations/tasks.py]: Would it be a good idea to create a named tuple for this instead of passing around a dict?

**Comment by cmc333333** [comment]
:+1: logging is my only real concern.

File: regulations/tasks.py (hunk 1 above)
Comment (vrajmohan) [inline regulations/tasks.py]: IMO, there's nothing sensitive. Here's an example:
```
[2016-03-24 17:15:12,896: WARNING/Worker-1] !!gov.egov.erule.regs.server.rest.response.CommentSuccessResponse
code: 201
developerMessage: Comment successfully created.
message: Comment successfully created.
moreInfo: N/A
status: 201
trackingNumber: 1k0-8oom-4uk7
uploadedFiles: [open_government0.pdf, open_government9.pdf]
[2016-03-24 17:15:12,896: WARNING/Worker-1] {'X-RateLimit-Remaining': '39', 'Via': 'http/1.1 api-umbrella (ApacheTrafficServer [cMs f ])', 'X-Cache': 'MISS', 'Content-Encoding': 'gzip', 'Age': '7', 'Vary': 'Accept-Encoding, Accept-Encoding, Accept-Encoding', 'Content-Length': '200', 'Server': 'openresty', 'Connection': 'keep-alive', 'X-RateLimit-Limit': '40', 'Cache-Control': 'max-age=600, public', 'Date': 'Thu, 24 Mar 2016 22:15:12 GMT', 'Content-Type': '*/*'}
[2016-03-24 17:15:12,899: INFO/MainProcess] Task regulations.tasks.submit_comment[1bbe24e5-8714-4e2e-bded-08e7ad174180] succeeded in 20.5541358681s: None
```

File: regulations/tasks.py (hunk 2 above)
Comment (vrajmohan) [inline regulations/tasks.py]: This is just a dict "reflected" from the JSON which forms the task payload. IMO, there is little to be gained from using a named tuple instead.

Comment (cmc333333) [inline regulations/tasks.py]: A namedtuple gets you closer to type safety, but I see your point. The extreme comes elsewhere in  -site, where a dictionary is the _primary_ data type (representing tree nodes).

**Pull request merged**